spec/controllers/activities_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe ActivitiesController, type: :controller do
+  include ControllerMacros
+  let(:user) do
+    create_user
+    User.last.update(admin: true)
+    User.last
+  end
+
+  describe "GET #index" do
+    it "returns http success if admin" do
+      user
```

Comment:
This syntax looks weird.

I guess there is a special case when we need user to be previosly created. What if we collect all the specs related to this case and move it into separate context. So we could create user (with `let` helper) for this context only.